src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -229,6 +229,40 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
                 }
             },
 
+            outerHTML: {
+                condition: domUtils.isElementNode,
+                get:       el => cleanUpHtml(el.outerHTML, el.parentNode && el.parentNode.tagName),
+
+                set: (el, value) => {
+                    var parentEl = el.parentNode;
+
+                    if (parentEl) {
+                        var parentDocument = domUtils.findDocument(parentEl);
+                        var parentWindow   = parentDocument ? parentDocument.defaultView : null;
+
+                        el.outerHTML = processHtml('' + value, parentEl.tagName);
+
+                        // NOTE: For the iframe with an empty src.
+                        if (parentWindow && parentWindow !== window &&
+                            parentWindow[INTERNAL_PROPS.processDomMethodName])
+                            parentWindow[INTERNAL_PROPS.processDomMethodName](parentEl, parentDocument);
+                        else if (window[INTERNAL_PROPS.processDomMethodName])
+                            window[INTERNAL_PROPS.processDomMethodName](parentEl);
+
+                        // NOTE: This check is required for an element in an unavailable window.
+                        // NOTE: Use timeout, so that changes take effect.
+                        if (window.self && domUtils.isBodyElement(el))
```

Comment:
Wow, that is a really twisted thing actually:
```js
document.head
//<head></head>

document.body
//<body></body>

document.head.outerHTML = '<head id="myhead"></head><body id="mybody"></body>';
//'<head id="myhead"></head><body id="mybody"></body>'

document.head
//<head id="myhead"></head>

document.body
//<body id="mybody"></body>

document.documentElement
//<html>
//    <head id="myhead"></head>
//    <body id="mybody"></body>
//    <body></body>
//</html>
```
Don't know, will be there a practical point of messing with that case?
I just need getter that cleans HTML to fix ASP tests.